src/mixed-line-bar-chart/hooks/use-navigation.ts

```diff
@@ -148,6 +137,15 @@ export function useNavigation<T extends ChartDataTypes>({
           series: nextSeries,
           datum: scaledSeries[scaledTargetIndex]?.datum,
         });
+      } else if (isXThreshold(nextSeries)) {
+        highlightSeries(nextSeries);
+        highlightPoint({
+          x: xScale.d3Scale(nextSeries.x as any) ?? NaN,
```

Comment:
Why the `any`?

Reply:
That's the limitation of our chart scales implementation. Those are not properly generic.